locator/host_id.hh

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (C) 2015-present ScyllaDB
+ *
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#pragma once
+
+#include "utils/UUID.hh"
+
```

Comment:
namespace locator?

I'm not crazy about it, but it seems appropriate.

Reply:
sure, not absolutely necessary, but it clarifies the scope.